src/generated/resources/openapi.yaml

```diff
@@ -1307,25 +1334,25 @@ components:
     HealthResponse:
       type: object
       properties:
-        checks:
-          type: array
-          items:
-            $ref: "#/components/schemas/HealthCheck"
         status:
           enum:
           - UP
           - DOWN
           type: string
+        checks:
+          type: array
+          items:
+            $ref: "#/components/schemas/HealthCheck"
     HealthCheck:
       type: object
       properties:
-        status:
-          enum:
-          - UP
-          - DOWN
+        name:
           type: string
         data:
           type: object
           nullable: true
-        name:
+        status:
```

Comment:
Gah.. I'll remove this in a bit.

Reply:
I'll revert this in the cleanup PR, I don't feel like waiting another 20 mins just for this.